Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectGenerator/MSBuildTemplates/SDKProjectTemplate.g.csproj.template

```diff
@@ -0,0 +1,14 @@
+<Project ToolsVersion="15.0">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(MSBuildForUnity.Common.props))"
+        Condition="Exists('$([MSBuild]::GetPathOfFileAbove(MSBuildForUnity.Common.props))')" />
+  <PropertyGroup>
+    <!--anborod: This is a weird thing, it is a required property (even if commented)-->
+    <!--<TargetFrameworks>netstandard2.0;uap10.0;net46</TargetFrameworks> -->
+  </PropertyGroup>
+
+  <Import Project="$(MSBuildForUnityGeneratedOutputDirectory)\$(MSBuildProjectName).g.props" />
+
+  <!--This file is generated and MAY BE OVERWRITTEN.-->
```

Comment:
Why is this comment down here on line 11? It seems like *every* generated MSBuild project file (.csproj, .g.props, .g.targets, etc.) should have a comment on line 1 indicating that it is a generated csproj, and whether it is ok to check it in or not (e.g. whether to expect to to be clobbered by MSBuildForUnity project generation).

Reply:
I am fine with putting this at the top of each of the generated files.